meedan/check-web
# Change Shared Feed Page Browser Tab Titles

## Description

Change Shared Feed Page Browser Tab Titles: 

- Landing page: Shared Feeds | Workspace Name
- Create page: Create Shared Feed | Workspace Name
- Edit page: Edit | Feed Name | Workspace Name
- List view: Feed Name | Workspace Name
- Item page: Item Title | Feed Name | Workspace Name

References: CV2-4431

## Type of change


## How has this been tested?

- Navigate to the landing page of the shared feeds.
Check that the browser tab title displays "Shared Feeds | Workspace Name".
- Navigate to the create page for a shared feed.
Check that the browser tab title displays "Create Shared Feed | Workspace Name".
- Navigate to the edit page of a shared feed.
Check that the browser tab title displays "Edit | Feed Name | Workspace Name".
- Navigate to the list view of a shared feed.
Check that the browser tab title displays "Feed Name | Workspace Name".
- Navigate to an item page within a shared feed.
Check that the browser tab title displays "Item Title | Feed Name | Workspace Name".

## Checklist
